### PR TITLE
Macro-free docs extra parens removed

### PR DESCRIPTION
**Pull request by anandijain**

### CODE REVIEW

File: docs/src/tutorials/symbolic_functions.md [hunk 1]
```diff
@@ -382,9 +382,9 @@ the user's code. For these cases, ModelingToolkit.jl allows for fully
 macro-free usage. For example:
 
 ```julia
-x = Sym{Float64}(:x)()
-y = Sym{Float64}(:y)()
-x+y^2.0
+x = Sym{Float64}(:x)
+y = Sym{Float64}(:y)
+x+y^2.0 # isa Num
```
Comment (shashi) [inline docs/src/tutorials/symbolic_functions.md:387]: It's not a Num actually

Comment (shashi) [reply docs/src/tutorials/symbolic_functions.md:387]: it would be a Num only if one of x or y was a Num

Comment (anandijain) [reply docs/src/tutorials/symbolic_functions.md:387]: I see, but given the prior two lines it makes more sense than nothing(?). I just tried `typeof(x+y^2.0)`.

I'm happy to remove it though

Comment (shashi) [reply docs/src/tutorials/symbolic_functions.md:387]: Yeah given the previous two lines we should probably have

```
x = Num(Sym{Float64}(:x))
```
etc.

And then `# isa Num` would make sense...

Comment (shashi) [reply docs/src/tutorials/symbolic_functions.md:387]: Let's do this, it's ugly but faithful

**Comment by anandijain** [comment]
Thanks, maybe this is not best practice, but re: your slack comment, I also changed `Expr(::Num)` because it errors, and `toexpr` doesn't

**Pull request merged**